client/src/components/HostViews/HostSubComponents/PausedDialog.vue

```diff
@@ -152,7 +150,7 @@ export default {
       return `
         ${
           window.location.href.substring(0, window.location.href.length - 5)
-        }/join?room=${
+        }/audience?room=${
```

Comment:
no, because that is the only place audience should join from. However if there are more use cases for it later, than that would certainly be a consideration

Reply:
That's fine with me